home/tests/test_add_answer.py

```diff
@@ -0,0 +1,73 @@
+from home.models import Profile, Answer
+import pytest
+from django.contrib.auth import authenticate
+from test_question_tag_model_functions import question_test_data
```

Comment:
question_test_data is a fixture defined in confest.py now not in test_question_tag_model_functions , so maybe this import is not relevant anymore